TechnicalTask-lvl1/Services/PersistenceService/CoreDataService.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  CoreDataService.swift
+//  TechnicalTask-lvl1
+//
+//  Created by Сергей Матвеенко on 25.11.24.
+//
+
+import Foundation
+import CoreData
+
+final class CoreDataService: CoreDataServiceProtocol {
+    
+    // MARK: - Parameters
+    
+    private var coreDataStack: CoreDataStack = {
```

Comment:
Навошта тут кложура? "крутость" не позволяет написать в одну строку?
```suggestion
    private var coreDataStack: CoreDataStack = CoreDataStack(modelName: CoreDataStrings.modelName)
```

Reply:
Почему напрямую устанавливается здесь, если остальная тушка намекает на переиспользование?